frontend/src/libs/components/link/link.tsx

```diff
@@ -6,10 +6,17 @@ import { type ValueOf } from '~/libs/types/types.js';
 type Properties = {
   to: ValueOf<typeof AppRoute>;
```

Comment:
why not like this ?
```suggestion
  to: AppRoute;
```

Reply:
it's was component from starter, and I just supplemented it. I didn't create a new wrapper component, so that there are no two similar ones.

Reply:
If indicate AppRoute I get an error, and need to indicate "typeof AppRoute", but than I have errors from all files where use prop to (because it's expected string, not object type). If I specify ValueOf<typeof AppRoute> work correct. So I leave it whiteout changes

Reply:
That's right, it should be `ValueOf<typeof AppRoute>;` because we use const instead of enums

Reply:
oh, sorry, I didn't think about it, thanks!